devel/eask-cli/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           npm 1.0
+PortGroup           elisp 1.0
+
+name                eask-cli
+version             0.8.3
+revision            0
+
+npm.rootname        @emacs-eask/cli
+distname            cli-${version}
+
+maintainers         {jcs090218 @jcs090218} openmaintainer
```

Comment:
the first entry here should be an obfuscated e-mail address; that doesn't seem correct here as you don't have a .....@macports.org e-mail address).

Reply:
👍